dcpy/connectors/edm/publishing.py

```diff
@@ -55,6 +55,13 @@ def path(self) -> str:
         return f"{self.product}/draft/{self.build}"
 
 
+@dataclass
+class GisDataset:
```

Comment:
Can name these something else - meant to refer to datasets in `edm-publishing/datasets`